framework/db/Migration.php

```diff
@@ -184,7 +189,13 @@ public function safeDown()
      */
     public function execute($sql, $params = [])
     {
-        echo "    > execute SQL: $sql ...";
+        $sqlPrint = $sql;
+
+        if ($this->printSqlLength !== 0 && mb_strlen($sql) > $this->printSqlLength) {
```

Comment:
Which one should I use?
I searched for a pattern in the framework and did not find it.
In fact, most cases are without encoding.